packages/common/src/services/ConfigService.ts

```diff
@@ -28,9 +28,7 @@ export default class ConfigService {
     content: [],
     menu: [],
     integrations: {},
-    styling: {
-      footerText: '',
```

Comment:
We need to keep footerText from here. It is still in app config, just not exposed in the JW dashboard. We can make the env values override it if you want.